src/Kestrel.Core/Internal/Http2/Http2StreamOfT.cs

```diff
@@ -22,18 +22,20 @@ public Http2Stream(IHttpApplication<TContext> application, Http2StreamContext co
             _application = application;
         }
 
-        public override async Task ProcessRequestAsync()
+        public override async Task ProcessRequestsAsync()
```

Comment:
Can we not share more code from this method? The error handling code in particular looks extremely complex and similar to FrameOfT.

Reply:
Agreed. I'll look into it.